spec/bookmarks-view-spec.coffee

```diff
@@ -114,43 +115,48 @@ describe "Bookmarks package", ->
         atom.commands.dispatch editorElement, 'bookmarks:toggle-bookmark'
         expect(bookmarkedRangesForEditor(editor).length).toBe 0
 
-    it "toggles proper classes on proper gutter row", ->
+    it "toggles proper classes on proper gutter and line row", ->
       editor.setCursorBufferPosition([3, 10])
       expect(editorElement.shadowRoot.querySelectorAll('.bookmarked').length).toBe 0
 
       atom.commands.dispatch editorElement, 'bookmarks:toggle-bookmark'
       lines = []
 
       waitsFor ->
-        lines = editorElement.shadowRoot.querySelectorAll('.bookmarked')
+        editorElement.shadowRoot.querySelectorAll('.line.bookmarked').length is 1
```

Comment:
Thank you for your keen eye, @jerone 
You're right, my specs were not catching problems related to `.line.bookmarked` items
It should be fixed now